tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "noImplicitOverride": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
+    "noImplicitAny": false,

```

Comment:
this was giving me guff trying to build, figured that at this point any any's we have right now are fine and we're unlikely to introduce a great deal more core functionality